docs/how-to/make-plots.md

```diff
@@ -0,0 +1,21 @@
+### Making plot using the outputs from `integrate`
+Since the outputs from `x.integrate` are matrices, you can use all normal MATLAB
+plotting functionality.
+
+### Making plots using the `plot` function
+The `x.plot` function will generate a figure, simulate the model, and plot the
+voltage traces. Here are a few important features to note:
+
+* If a figure handle is set up in `x.handles.fig` it will use it. Otherwise, it will
+generate the plot and put the figure handle there.
+* Subplot handles are stores in `x.handles.ax`.
+* The plots are colored by the largest contributing current. You can turn this
+functionality off by setting `setpref('xolotl', 'plot_color', false)`.
```

Comment:
no, not anymore. 

this is in x.pref.plot_color (temporary)

or for a more permanent setting, change `pref.m`